### PR TITLE
[RC1] Rename neutron-tempest-plugin-api to tempest_neutron

### PR DESCRIPTION
**Pull request by collivier**
It unifies the test case names.
It completes the changes on Functest and Releng [1]

[1] https://gerrit.opnfv.org/gerrit/c/functest/+/70016

Signed-off-by: Cédric Ollivier <cedric.ollivier@orange.com>

### CODE REVIEW

File: doc/ref_impl/cntt-ri/chapters/chapter08.md [hunk 1]
```diff
@@ -447,44 +447,44 @@ skipping tests if optional services (e.g. Gnocchi, Barbican, etc.) or
...
-| opnfv/functest-benchmarking-cntt:hunter | vmtp                       | Success            |
-| opnfv/functest-benchmarking-cntt:hunter | shaker                     | Success            |
-| opnfv/functest-vnf:hunter               | cloudify                   | Success            |
-| opnfv/functest-vnf:hunter               | cloudify_ims               | Success            |
-| opnfv/functest-vnf:hunter               | heat_ims                   | Success            |
-| opnfv/functest-vnf:hunter               | vyos_vrouter               | Success            |
-| opnfv/functest-vnf:hunter               | juju_epc                   | Success            |
+| container                               | test case        | criteria (Jenkins) |
+|-----------------------------------------|------------------|:------------------:|
+| opnfv/functest-smoke-cntt:hunter        | tempest_neutron  | Success            |
```
Comment (pgoyal01) [inline doc/ref_impl/cntt-ri/chapters/chapter08.md:472]: should the test cases not be tempest-neutron-cntt, tempest-cinder-cntt, ... for the first seven cases?

**Comment by pgoyal01** [comment]
The following suggestion is not for this PR. Suggest adding a column "Requirements" to [3.4.1 RM/RA-1 Requirements](https://github.com/cntt-n/CNTT/blob/master/doc/ref_cert/lfn/chapters/chapter03.md#341-rmra-1-requirements) -- this will allow 3.4.2 to be deleted.

File: doc/ref_impl/cntt-ri/chapters/chapter08.md (hunk 1 above)
Comment (collivier) [reply doc/ref_impl/cntt-ri/chapters/chapter08.md:472]: You're right. I should have changed them in my previous change.
Fixing here. Thank you!

**Comment by collivier** [comment]
@pgoyal01 a few Functest test cases (tempest_full) will cover all requirements related to OpenStack operations (e.g. related to VM and disk)

**Comment by rabi-abdel** [comment]
@fuqiao123 @jayacha may you please review

**Pull request merged**